apisix/plugins/limit-count/limit-count-redis.lua

```diff
@@ -71,7 +71,8 @@ function _M.incoming(self, key)
     local remaining
     key = self.plugin_name .. tostring(key)
 
-    local ret = red:ttl(key)
+    -- todo: test case
+    local ret = red:ttl(key) or -2
```

Comment:
I think we need to cache the `err`, eg: `local ret, err = red:(...)`

And we should write this error message to log, set the right response like `500`.

Reply:
OK, fixed.